src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -216,13 +227,36 @@ typedef struct uct_rc_mlx5_iface_common {
     UCS_STATS_NODE_DECLARE(stats);
 } uct_rc_mlx5_iface_common_t;
 
+/**
+ * RC mlx5 interface configuration
+ */
+typedef struct uct_rc_mlx5_iface_config {
+    uct_rc_iface_config_t             super;
+    uct_rc_fc_config_t                fc;
+    uct_ib_mlx5_iface_config_t        mlx5_common;
+    unsigned                          tx_max_bb;
+    /* TODO wc_mode, UAR mode SnB W/A... */
+} uct_rc_mlx5_iface_config_t;
```

Comment:
is it common config? then need to add `common` to the name.
Also `uct_rc_fc_config_t` is not needed for DC, so would need to move to RC mlx5 only